src/ralph/attachments/views.py

```diff
@@ -7,6 +7,19 @@
 from ralph.helpers import add_request_to_form
 
 
+def get_attachment_url_name(model, with_namespace=True):
```

Comment:
We already have a similar method in the MulitiAddAdminMixin(get_url_name) can write generic function?